site-src/blog/2022/first-beta.md

```diff
@@ -0,0 +1,142 @@
+---
```

Comment:
Our blog system is remarkably manual at this point, so we'll also need additions to https://github.com/kubernetes-sigs/gateway-api/blob/master/site-src/blog/index.md and https://github.com/kubernetes-sigs/gateway-api/blob/master/mkdocs.yml#L69 for this to fit in.

Reply:
Sounds good, done in 82421d13bd444b49980d66e8db97824026d5d053